description.txt

```diff
@@ -1 +1,19 @@
-This is a description file for git training session, now resolve conflicts 
\ No newline at end of file
+This is a description file for git training session, now resolve conflicts
```

Comment:
@akshay-mundra The branch is not proper, it should depict what changes you have done.
Going forward, kindly keep the branch name properly  it could be `docs-update-session-mom`